src/main/java/org/example/homework21apiglovodb/dto/ItemDto.java

```diff
@@ -0,0 +1,14 @@
+package org.example.homework21apiglovodb.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ItemDto {
```

Comment:
Maybe better to use only one model(ItemEntity and OrderEntity) because this classes have the same data

Reply:
Can't say so, they supposed to be a separate as entities are stored objects in the db